ftw/simplelayout/contenttypes/profiles/default/jsregistry.xml

```diff
@@ -8,4 +8,11 @@
         insert-after="++resource++ftw.simplelayout.resources/main.js"
         />
 
+    <javascript
+        id="++resource++ftw.simplelayout.resources/RequestTracker.js"
```

Comment:
What is the reason you placed that in the `contenttypes` package?
I think that this is useful regardless of whether the contenttypes are installed.
I would move it to the `lib` profile.